maxim/logger/crewai/client.py

```
@@ -582,6 +634,7 @@ def maxim_wrapper(self, *args, **kwargs):
                         )
                         for tool in tools_to_set_context:
                             setattr(tool, "_span", span)
+                            setattr(tool, "_eval_config", eval_config)
```

Ruff (0.12.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)